scripts/issueEmailGenerator.js

```diff
@@ -84,6 +85,21 @@ if (typeof options.issueNumber !== 'number') {
     return `${process.env.NEXT_PUBLIC_ASSETS_URL}issue-${issueNumber}/${assetURL}`;
   }
 
+  function getAuthors(authors) {
+    if (authors?.length) {
+      console.log(authors);
+      const authorsWithWebsite = authors.map((author, index) => {
+        const isLastElement = index === authors.length - 1;
+        console.log(isLastElement);
+        return `[${author.Name}](${author.Website})` + (isLastElement ? '' : `, `);
+      });
+
+      return `*by ${authorsWithWebsite}*`;
```

Comment:
And then joining like this should do the job I believe
```suggestion
      return `*by ${authorsWithWebsite.join(',')}*`;
```

Reply:
The VS Code Markdown Preview was confusing me as it was showing me two commas, but in reality, there was only one. 😏